src/Acuminator/Acuminator.Analyzers/StaticAnalysis/LegacyBqlField/LegacyBqlFieldAnalyzer.cs

```diff
@@ -25,23 +25,29 @@ internal override void AnalyzeCompilation(CompilationStartAnalysisContext compil
 
 		private async void Analyze(SymbolAnalysisContext context, PXContext pxContext)
 		{
-			if (context.Symbol is INamedTypeSymbol dacFieldType)
+			if (context.Symbol is INamedTypeSymbol dacOrDacExt)
```

Comment:
@Derrekam the nested sope becames too big. We have a code style rule that indentation depth shouldn't be more than 3:
https://github.com/Acumatica/Acuminator/blob/dev/docs/dev/CodingGuidelines/CodingGuidelines.md#indentation-depth

I would suggest to reverse big if clause since it's scope is the whole method:
```
if (!(context.Symbol is INamedTypeSymbol dacOrDacExt))
    return;
```
Actually you can merge it with the next if:
```
if (!(context.Symbol is INamedTypeSymbol dacOrDacExt) || (!dacOrDacExt.IsDAC() && !dacOrDacExt.IsDacExtension()) || context.CancellationToken.IsCancellationRequested)
					return;
```

Reply:
done